py3status/modules/getjson.py

```diff
@@ -94,7 +94,7 @@ def _query_url(self):
             resp = self.py3.request(self.url, timeout=self.timeout)
             status = resp.status_code == 200
             resp = resp.json()
-        except self.Py3.RequestException:
+        except self.py3.RequestException:
```

Comment:
@maximbaz good catch catch.

If you add this as a separate PR I'll merge it straight away as a bugfix